.github/workflows/ci.yml

```diff
@@ -8,12 +8,10 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
-        node-version: [8.x, 9.x, 10.x, 11.x, 12.x, 13.x]
+        node-version: [8.x, 9.x, 10.x, 12.x, 14.x]
```

Comment:
What is the reason for keeping 8 & 9 around?

Reply:
Because we technically still have node 8 and 9 compatibility, so I don't want to remove them until we maybe add a breaking change and say we don't support those versions anymore.

Reply:
Got it. Fwiw both of those versions of node no longer receive any support.